process/coordinator/process.go

```diff
@@ -563,6 +563,7 @@ func (tc *transactionCoordinator) CreateMarshalizedData(body block.Body) (map[ui
 		if receiverShardId == tc.shardCoordinator.SelfId() { // not taking into account miniblocks for current shard
```

Comment:
There is a method VerifyCreatedBlockTransactions which does not have a comment

Reply:
done